base/build-static-ioc.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -ex
+
+IOC=$1
+
+cp /opt/epics/RELEASE configure/
+cat << EOF > configure/CONFIG_SITE.local
+STATIC_BUILD=YES
+FULL_STATIC_BUILD=YES
+USR_LDFLAGS = -static-pie
+EOF
+
+git config --global --add safe.directory $PWD
+
+make_skip() {
+  if [ -z "$SKIP_CLEAN" ]; then
+    make "$@"
+  fi
+}
+
+make_skip distclean
+make
+make_skip clean
+make -C iocBoot
+
+version=$(git describe --tags)
```

Comment:
NIT: this command add the number of additional commits and an abbreviated object name for the commit itself at the end of the tag. Maybe using the flag `--abbrev=0` keeps the tarball name more clean.

Reply:
In this case I do want the object name. If it's a tag, the name is clean, if it's  a development version, we have all the necessary information about the version.